### ISSUE TITLE
HELP controller disconnected, no computer input!

### ISSUE THREAD

**Issue by mcmoodie** [Ryochan7/DS4Windows#1197]
I managed to finally convince my controller to connect, and after having some general game issues on Yandere Simulator related to its debug version, I disconnected to see if controller mappings were the problem. I unplugged my wired connection and my computer will no longer recognize it's own trackpad, but does recognize the keyboard,(Toshiba Laptop). Using the touchpad on the controller, I was able to navigate back to the Ds4 Windows page and I clicked STOP to see if that would work. Now there is nothing I can use to input anything, and I cannot figure out how to resume DS4 or use the trackpad.

Edit
After many tries the issue seems to be that DS4 wont stop after I've told it to stop. The laptop only recognizes the touchpad of my Ps4 controller as the trackpad, with buttons assigned as in  DS4 settings. Once unplugged, there is no longer a working mouse function.

**Comment by mcmoodie** [comment]
Let me know if I need to describe anything in any more detail but I really need this fixed soon please.

**Comment by mcmoodie** [comment]
Most current version of DS4 Windows, just downloaded it today (April 1st)

**Comment by mcmoodie** [comment]
I force restarted the laptop and that seems to have fixed it until I actually logged in. I dont think DS4 has permission to boot on launch.

**Comment by mcmoodie** [comment]
After much trial and error I was able to navigate to ds4 windows and reconnect my controller to it.
![15857803315561881521886642077996](https://user-images.githubusercontent.com/63020927/78192894-21652c00-7447-11ea-8bd2-dbcc90910617.jpg)
Here's my log for this startup

**Comment by mcmoodie** [comment]
FIXED

I wasnt able to fix it. I used a Factory Restore point and restored it to right before I downloaded the secondary part of the Driver. Uninstalled the whole program, wont use again for a while.

**Comment by mika-n** [comment]
Hmm... Don't know how DS4Windows or ViGem driver could even theoretically cause something like this. 
Are you using HidGuardian or had you previously used an old Jays2Kings version of DS4Windows with ScpToolkit DS4 drivers?  Theoretically HidGuardian could cause something like this if installation of HidG is screwed. Or the old ScpToolki/ScpvBus driver may cause conflicts with ViGem driver, but still it shouldn't cause anything like you described.

Some laptops have an automatic functionality to disable touchpad when an external mouse is connected (standard usb mouse). For example my laptop behaves like this by default. However, I have set an option in Settings/Touchpad to leave the touchpad enabled even when an external mouse is connected. Maybe you wanna check this WinOS Settings/Touchpad option.

 Did you try to use external usb mouse when the touchpad was not working?

Anyway. Good thing things are now sorted out, but don't know how DS4Windows+VigEm could even theoretically cause something like this.

**Comment by mcmoodie** [comment]
I'm not the most tech savvy guy, but I was thinking if I did have an external usb mouse it would probably fix it. I didnt download anything but the github link to 64 bit DS4 2.0.14 (I think), and the install of VigEm. I didnt configure anything, or mess with the settings just in case of something happening that I didnt know how to fix. I dont really know what happened. I think the laptop is older, and maybe that had something to do with it. My boyfriend uses the program no problem so I know it's not that I got screwed on something. Thank you for responding though, I dont really want to mess with anything so I'll just wait to do anything until I get my own laptop so I dont have to apologize to my mom lol
